lib/ratchet/component.js

```diff
@@ -20,10 +20,10 @@ function createChildren(node, data) {
 }
 
 function createComponent(node, data) {
-  var attrs = attrsToProps(node.attributes);
```

Comment:
I was also trying to keep with the react terminology which is where this data ultimately ends up. DOM has attributes, React components have properties.